wordpress-mobile/AztecEditor-Android
# Issue/367 layout and order

### Fix
Update format toolbar layout and order of the format buttons as shown below and as described in https://github.com/wordpress-mobile/AztecEditor-Android/issues/367.

![toolbar_expanded](https://user-images.githubusercontent.com/3827611/26945887-97273ffa-4c4a-11e7-96ec-c759751eb7e2.png)

### Test
1. Run demonstration app.
2. Notice format toolbar appears like screenshot above.
